### PR TITLE
Beautify python

### PR DESCRIPTION
**Pull request by gczsjdy**

### CODE REVIEW

**Comment by carsonwang** [comment]
can you remove the changes that has already been merged in #373 ?

File: bin/functions/load_config.py [hunk 1]
```diff
@@ -564,35 +551,34 @@ def probe_masters_slaves_hostnames():
...
-                    # Make the assumption that the master is in internal network, and force not to use any proxies
+                    # Make the assumption that the master is in internal network, and force
+                    # not to use any proxies
                     with closing(urllib.urlopen('http://%s:%s' % (HibenchConf['hibench.masters.hostnames'], master_port), proxies={})) as page:
                         worker_hostnames = [
                             re.findall(
                                 "http:\/\/([a-zA-Z\-\._0-9]+):%s" %
                                 worker_port,
                                 x)[0] for x in page.readlines() if "%s" %
                             worker_port in x and "worker" in x]
```
Comment (carsonwang) [inline bin/functions/load_config.py]: Can you also update the above lines?

Comment (gczsjdy) [inline bin/functions/load_config.py]: Do you mean line 563~569? It's the version updated.

**Comment by gczsjdy** [comment]
Done, is there any other code that need update? @carsonwang

**Comment by carsonwang** [comment]
Looks good. Merging.

**Pull request merged**